strings/shortest_word.py

```diff
@@ -0,0 +1,28 @@
+def shortest_word(sentence: str) -> str:
+    """
+    Find the shortest word in a sentence.
+    If there are multiple words that are the same length, return the first one.
+
+    www.geeksforgeeks.org/python-program-to-find-the-smallest-word-in-a-sentence/
+
+    >>> shortest_word("I love Python")
+    'I'
+    >>> shortest_word("Find the shortest word in a sentence.")
+    'a'
+    >>> shortest_word("Black Stack")
+    'Black'
+    >>> shortest_word("funny bunny rabbit")
+    'funny'
+    """
+
+    # Split the sentence into a list of words
+    # Sort by length and take the first element
+    word_list = sentence.split()
+    word_list.sort(key=len)
```

Comment:
Sorting requires N*logN complexity,  you could get the same result just in N by iterating over the sentence and finding the min length.   min(word_list , key=len) can do the trick.